sputnikdao2/src/types.rs

```diff
@@ -1,11 +1,14 @@
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::Base64VecU8;
 use near_sdk::serde::{Deserialize, Serialize};
-use near_sdk::{Balance, Gas};
+use near_sdk::{AccountId, Balance, Gas};
 
-/// Account ID used for $NEAR.
-pub const BASE_TOKEN: &str = "";
+/// Account ID used for $NEAR in near-sdk v3.
+/// Need to keep it around for backward compatibility.
+pub const OLD_BASE_TOKEN: &str = "";
```

Comment:
@ctindogaru consider `V2_FALLBACK_TOKEN` or similar to denote when and what this is for, so future can be removed.

Reply:
BASE_TOKEN is like a convention in Sputnik, used for $NEAR.

See also https://github.com/near-daos/sputnik-dao-contract/blob/main/sputnikdao2/tests/utils/mod.rs#L27-L29.

So to keep this consistent, I would stick with BASE_TOKEN :D